src/applications/vaos/tests/new-appointment/components/calendar/CalendarWidget.unit.spec.jsx

```diff
@@ -342,109 +330,4 @@ describe('VAOS <CalendarWidget>', () => {
       expect(tree.instance().isValid(getDateFromButtonId(button))).to.be.true;
     });
   });
-
```

Comment:
I'm fine with removing these tests, but we need to make sure that each of these behaviors are being replicated in our RTL tests.  Can you confirm that they are?

Reply:
I was debating doing this in a separate ticket (since I want to remove all these widget tests eventually), but we could also add some of the assertions from these to the RTL tests now.

It looks to me like it would mainly be checking that a date is properly saved in the Redux state when you pick one.